Tests/PackageLoadingTests/PD4_2LoadingTests.swift

```diff
@@ -182,7 +182,6 @@ class PackageDescription4_2LoadingTests: XCTestCase {
                    .package(path: "~/path/to/foo10"),
                    .package(path: "~foo11"),
                    .package(path: "~/path/to/~/foo12"),
-                   .package(path: "~/"),
```

Comment:
If home directory just returns `/`, this path is resolved as `/`.
However, SPM currently not supports `/` as a dependency URL.
So, I dropped this case.